core/src/Cuda/Kokkos_Cuda_Graph_Impl.hpp

```diff
@@ -200,6 +200,9 @@ struct GraphImpl<Kokkos::Cuda> {
         m_execution_space, _graph_node_kernel_ctor_tag{},
         aggregate_kernel_impl_t{});
   }
+
+  const cudaGraph_t get_native_graph() const { return m_graph; }
+  const cudaGraphExec_t get_native_executable_graph() const { return m_graph_exec; }
```

Comment:
I am tempted to suggest to call these `get_cuda_graph[_exec]` because I am not convinced there is much generic code one could write that get the native graph nor that the concept of "executable" graph will always make sense.

Reply:
- Agreed for `_exec` suffix.
- For the `get_cuda_graph`, it would mean `get_cuda_graph`/`get_hip_graph`, and so on (*i.e.* not portable across backends). I guess the question is if you want something that looks like `space.cuda_stream()`/`space.hip_stream`, or if you want something like `get_graph_impl` ("portable" at least across `Cuda` and `HIP` backends)?

Reply:
That's my point, it is more about interoperability than portability here.

Reply:
Fair enough !